codex/stores/networkstore.nim

```diff
@@ -32,17 +32,17 @@ type
     localStore*: BlockStore # local block store
 
 method getBlock*(self: NetworkStore, cid: Cid): Future[?!bt.Block] {.async.} =
-  ## Get a block from a remote peer
-  ##
-
   trace "Getting block from local store or network", cid
 
   without blk =? await self.localStore.getBlock(cid), error:
     if not (error of BlockNotFoundError): return failure error
     trace "Block not in local store", cid
-    # TODO: What if block isn't available in the engine too?
```

Comment:
Are we handling this case? The first TODO seems important, unless it's already handled elsewhere?